src/main/java/com/slickqa/webdriver/DefaultWebDriverWrapper.java

```diff
@@ -217,14 +217,17 @@ public void setCheckboxState(PageElement locator, boolean checked, int p_timeout
                     break;
                 }
             } catch (Exception e) {
```

Comment:
change Exception to RuntimeException, then you don't have to have a wrapper around the exception